carmin-server/server/resources/execution.py

```diff
@@ -1,9 +1,23 @@
 from flask_restful import Resource
+from server.database.queries.executions import get_execution
+from server.common.error_codes_and_messages import (
+    ErrorCodeAndMessageFormatter, EXECUTION_NOT_FOUND)
```

Comment:
This is a bit out of scope, but I think we should include the provided Execution identifier as part of the `EXECUTION_NOT_FOUND` ErrorCodeAndMessage.